src/core/controllers/oidc.go

```diff
@@ -85,7 +87,12 @@ func (oc *OIDCController) Callback() {
 		oc.SendBadRequestError(errors.Errorf("OIDC callback returned error: %s - %s", errorCode, errorDescription))
 		return
 	}
-
+	var redirectUrlStr string
+	redirectUrl := oc.GetSession(redirectUrlKey)
```

Comment:
var redirectUrl should be redirectURL (revive)
see: https://github.com/goharbor/harbor/actions/runs/3255311148/jobs/5344498524